src/hardware/RX/Frank 2400.json

```diff
@@ -0,0 +1,35 @@
+{
+    "serial_rx": 3,
+    "serial_tx": 1,
+    "radio_busy": 36,
+    "radio_dio1": 39,
+    "radio_miso": 38,
+    "radio_mosi": 23,
+    "radio_nss": 10,
+    "radio_rst": 9,
+    "radio_sck": 18,
+    "power_min": 0,
+    "power_high": 0,
+    "power_max": 0,
+    "power_default": 0,
+    "power_control": 0,
+    "power_values": [13],
+    "led_rgb": 2,
+    "led_rgb_isgrb": 1,
+    "led_rgb_status": [0],
+    "led_rgb_vtx": [1],
+    "led_rgb_boot": [0,1],
+    "vtx_nss": 15,
+    "vtx_miso": 37,
+    "vtx_mosi": 13,
+    "vtx_sck": 14,
+    "rf_amp_pwm": 25,
+    "rf_amp_vpd": 35,
+    "rf_amp_vref": 26,
```

Comment:
I think we need to add vtx_ e.g. `vtx_rf_amp_pwm`, so that they can not be confused in future with the 2.4 rf amp.

Reply:
Maybe `vtx_amp_???`

Reply:
I renamed `rf_amp_` to `vtx_amp_` and also the `vpd_` to vtx_amp_vpd_` since they are all related to the VTX.

Reply:
Good call on adding it vpd_ as well.